src/app/features/shows/pages/movie-details/movie-details.component.html

```diff
@@ -1 +1,41 @@
-<p>movie-details works!</p>
+<div *ngIf="loading; else showMovieDetails">
+  <app-movie-details-skeleton></app-movie-details-skeleton>
+</div>
+<ng-template #showMovieDetails>
+  <div class="heading">
+    <img src="https://image.tmdb.org/t/p/original/{{ movie.backdropPath }}" alt="backdrop" class="backdrop">
```

Comment:
```suggestion
    <img src="https://image.tmdb.org/t/p/w1280/{{ movie.backdropPath }}" alt="backdrop" class="backdrop">
```
hình gốc nặng gấp mấy lần hình mình cần á e

Reply:
đặt min-height để tránh bị hiện tượng nhảy element